health-services/ExerciseSampleCompose/gradle/libs.versions.toml

```diff
@@ -1,5 +1,5 @@
 [versions]
-android-gradle-plugin = "8.6.0"
+android-gradle-plugin = "8.6.0-alpha05"
```

Comment:
?

Reply:
Me and my version of Android Studio... adjusted back